modules/sc-mesh-secure-deployment/src/gw/main.py

```diff
@@ -38,15 +38,18 @@ def __init__(self, init_gw_mode: str="server"):
     def clean_and_start(self):
         return_code, null = run_shell_command(
             f"sysctl net.ipv4.ip_forward=1")
+        print("Log 1 for return_code should be !0:" + return_code)
```

Comment:
Lot of print() - calls added even the module uses logger for logging?  Are these really neeeded?

Reply:
Agree, this should go to a log file

Reply:
These were for testing purposes, forgot to remove them!

Reply:
Yes, those looks like a debug/testing prints and not needed for the final gw logging.